### PR TITLE
Read method for Quizzes

### PR DESCRIPTION
**Pull request by AnyaMaz**
#368 
- [X] Add Read method
- [X] Add tests for method
- [X] Add acl

### CODE REVIEW

**Comment by AnyaMaz** [comment]
check pr plz

File: src/main/java/easytests/api/v1/controllers/QuizzesController.java [hunk 1]
```diff
@@ -63,4 +63,26 @@
     /**
      * show(quizId)
      */
+
+    @GetMapping("/{quizId}")
+    public Quiz show(@PathVariable Integer quizId) throws NotFoundException, ForbiddenException {
+        final QuizModelInterface quizModel = this.getQuizModel(
```
Comment (malinink) [inline src/main/java/easytests/api/v1/controllers/QuizzesController.java]: nope, you shold use forAuth to fetch Quiz, that will be good for hasAcces mathod call.
ForAuth should be implemented in `getQuizModel(Integer id)` if there is no such method yet.

File: src/test/java/easytests/api/v1/controllers/QuizzesControllerTest.java [hunk 2]
```diff
@@ -150,4 +151,95 @@ public void testListForbidden() throws Exception {
...
+                .andExpect(content().string(""))
+                .andReturn();
+    }
+
+    @Test
+    public void testShowWithTesteeSuccess() throws Exception {
+        final QuizModelInterface quizModel = new QuizModel();
+        quizModel.map(this.quizzesSupport.getEntityFixtureMock(0));
+        final TesteeModelInterface testeeModel = new TesteeModel();
+//        final List<TesteeModelInterface> testeesModels = new ArrayList<>();
```
Comment (malinink) [inline src/test/java/easytests/api/v1/controllers/QuizzesControllerTest.java]: remove unused code from PR

**Review by malinink — CHANGES_REQUESTED** [review]

**Comment by AnyaMaz** [comment]
@malinink Rdy For Review

**Review by malinink — APPROVED** [review]

**Pull request merged**